packages/pages/src/common/src/project/structure.ts

```diff
@@ -31,6 +31,8 @@ export interface Subfolders {
   redirects: string;
   /** The Node functions folder */
   serverlessFunctions: string; // Node functions
+  /** The puck folder */
+  puck: string;
```

Comment:
We shouldn't have anything specific to Puck in the pagesjs code (other than creating these VE templates/puckConfig). "puck" as a folder in project structure shouldn't be necessary. Also, everything in there now will be part of the new visual-editor lib and a user would just have an edit.tsx template that uses the <Edit> component we export from that lib. I know it doesn't exist yet, but that's what this PR should make use of.